src/tools/coverage-dump/src/covfun.rs

```diff
@@ -70,10 +70,13 @@ pub(crate) fn dump_covfun_mappings(
                     }
                     // If the mapping is a branch region, print both of its arms
                     // in resolved form (even if they aren't expressions).
-                    MappingKind::Branch { r#true, r#false } => {
+                    MappingKind::Branch { r#true, r#false }
+                    | MappingKind::MCDCBranch { r#true, r#false, .. } => {
                         println!("    true  = {}", expression_resolver.format_term(r#true));
                         println!("    false = {}", expression_resolver.format_term(r#false));
                     }
+
+                    MappingKind::MCDCDecision { .. } => {}
```

Comment:
This case isn't needed, as it will be caught by the `_` case anyway and do nothing.

Reply:
Fix at [e6fce0b8d](https://github.com/rust-lang/rust/pull/123409/commits/e6fce0b8d4995b0050e75069b0f83f580e7afecd)